app/Http/Controllers/ReadingItemController.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\ReadingItem;
+use Illuminate\Http\Request;
+
+class ReadingItemController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        return view('reading-items.index')->with([
+            'reading_items' => ReadingItem::all()->sortByDesc('created_at'),
+        ]);
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        //
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        ReadingItem::create([
+            'topic' => $request->input('topic'),
+            'url' => $request->input('url'),
+            'recommended_by' => $request->input('recommended_by'),
+        ]);
+        return json_encode(['success' => true]);
```

Comment:
Can we create a Response ? I see that this response is commonly used. Can you create a response macros for this ?

https://laravel.com/docs/5.6/responses#response-macros